src/packages/admin-ui-components/src/detail-panel/component.tsx

```diff
@@ -266,6 +265,7 @@ export const DetailPanel = () => {
 					variables: {
 						data: values,
 					},
+					refetchQueries: [`AdminUIListPage`],
```

Comment:
Should we be doing this refetch everywhere in our examples? My s3 example doesn't refetch and therefore doesnt show a newly created submission until the page is refreshed.

Reply:
This will fix it for all creates including your example for S3.